src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -198,7 +198,7 @@ public bool TryParseEofHeader(ReadOnlySpan<byte> container, [NotNullWhen(true)]
                 {
                     if (_loggingEnabled)
                         _logger.Trace($"EIP-3540 : Empty Code Section are not allowed, CodeSectionSize must be > 0 but found {codeSection.Size}");
-                    return false;
+                    header = null; return false;
```

Comment:
Cosmetic thing: why is header = null and return in the same line?

Reply:
Error from my side, I used `replace all` command without adding a new line